scripts/mina-sync-monitor.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e
+exit_code=1
```

Comment:
**nitpick:** I suggest to remove this `exit_code` var, and use directly `exit 0` or `exit 1` where required.

```suggestion
```

Reply:
Code updated